bin/pnpm

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env sh
+
+if [ "$#" -lt "1" ]; then
+  echo "You should pass at least 1 parameter to pnpm" 1>&2
```

Comment:
this should probably eventually show a help screen like `npm --help`... but for another time; for now this is gold